src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -1181,9 +1137,7 @@ internal static bool IsDefined(RuntimeConstructorInfo ctor, RuntimeType caType)
             Debug.Assert(ctor != null);
             Debug.Assert(caType != null);
 
-            if (PseudoCustomAttribute.IsDefined(ctor, caType))
-                return true;
-
+            // No pseudo attributes for RuntimeConstructorInfo
```

Comment:
Nit: It would be nice to have an empty new line after this comment, like it is for RuntimeAssembly above, to make it clear that the comment is not for `IsCustomAttributeDefined` call.

(Multiple places)